app/services/analytics_events.rb

```diff
@@ -962,8 +962,15 @@ def idv_proofing_resolution_result_missing
   end
 
   # User submitted IDV password confirm page
-  def idv_review_complete
-    track_event('IdV: review complete')
+  # @param [Boolean] success
+  def idv_review_complete(success:, **extra)
+    track_event(
+      'IdV: review complete',
+      {
+        success: success,
+        **extra,
+      }.compact,
```

Comment:
Is the `compact` here necessary? Seems the same as what we're doing in other events?

```suggestion
      success: success,
      **extra,
```

Reply:
Probably not. This was picked up from a copy/paste from some other events in here. I can drop it.